pyaerocom/aeroval/experiment_processor.py

```diff
@@ -101,8 +106,20 @@ def run(self, model_name=None, obs_name=None, var_list=None, update_interface=Tr
             var_list = [var_list]
 
         self.cfg._check_time_config()
-        model_list = self.cfg.model_cfg.keylist(model_name)
+
         obs_list = self.cfg.obs_cfg.keylist(obs_name)
+        if self.cfg.model_cfg == {}:
```

Comment:
you could use the fact that an empty dictionary "is falsy"
``` python
        if not self.cfg.model_cfg:
            ...
```
this also covers the case when `self.cfg.model_cfg is None`

Is it possible for the `model_cfg` attribute to be missing?

Reply:
I don't think it is possible. I think aeroval will crash if no model_cfg is given

Reply:
and `model_cfg = None`?